src/main/java/uk/gov/hmcts/reform/divorce/orchestration/controller/CaseController.java

```diff
@@ -61,27 +61,22 @@
 
     }
 
+    @SuppressWarnings("unchecked")
     @PatchMapping(path = "/case", consumes = APPLICATION_JSON, produces = APPLICATION_JSON)
     @ApiOperation(value = "Patch case in CCD")
     @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "Update was successful and case was updated in CCD",
-            response = CaseResponse.class),
-        @ApiResponse(code = 400, message = "Bad Request")})
-    public ResponseEntity<CaseResponse> updateCase(
+        @ApiResponse(code = 200, message = "Update was successful and case was updated in CCD"),
+        @ApiResponse(code = 400, message = "Json payload has missing field")})
+    public void updateCase(
         @RequestHeader(value = AUTHORIZATION_HEADER) String authorizationToken,
-        @RequestBody @ApiParam("Divorce Session") Map<String, Object> payload) throws WorkflowException {
-
-        Map<String, Object> serviceResponse = caseService.patchCase(payload, authorizationToken);
+        @RequestBody @ApiParam("Divorce Session") Map<String, Object> payload) {
 
-        if (serviceResponse.containsKey(VALIDATION_ERROR_KEY)) {
-            log.error("Bad request. Found this validation error: {}", serviceResponse.get(VALIDATION_ERROR_KEY));
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).build();
-        }
+        final Object id = Optional.ofNullable(payload.get("id"))
+            .orElseThrow(() -> new BadRequestException("Missing field 'id' in json payload."));
```

Comment:
can we reuse `HttpClientErrorException` instead of creating this new Exception ?

Reply:
I went for ResponseStatusException as it is easier to create, and removed BadRequestException